src/Lkl.cc

```diff
@@ -483,6 +486,7 @@ void Lkl::FindGLowAndGUpp(Double_t& glow,Double_t& gupp,Bool_t centerAtZero)
       Double_t save  = gupp;      
       gupp-=TMath::Abs(gupp-gmin)/2.;
       cout << "Lkl::FindGLowAndGUpp (" << GetName() << ") Message: -2logL(gupp) = " << lklval << ", gupp lowered from " << save << " to " << gupp <<" (fLklMin = " << fgmin << ")"<< endl;
+      if(lklval >= 1e99) break;
```

Comment:
This can never happen because of the loop condition, right?

Reply:
What in the loop condition prevents it?

Reply:
No, I mean like before, that this should not generate an infinite loop. I believe you when you say it does, but I'd like to understand how, again I'm gonna need the example...